src/index.test.ts

```diff
@@ -6,6 +6,57 @@ const fontWeightKeywords = require('css-font-weight-keywords');
 const fontStyleKeywords = require('css-font-style-keywords');
 const fontStretchKeywords = require('css-font-stretch-keywords');
 
+const numberValues = [
```

Comment:
I think you'll find that many of these tests cases are covered [here](https://github.com/jedmao/parse-css-dimension/blob/master/test/index.js), so as not to reinvent the wheel.